doc/guides/writing-tests.md

```diff
@@ -18,6 +18,15 @@ Add tests when:
 - Fixing regressions and bugs.
 - Expanding test coverage.
 
+## Test directory structure
+See the test [directory structure overview](https://github.com/nodejs/node/blob/master/test/README.md)
```

Comment:
Adding a new line after the header would be good for consistency as well.

Reply:
changes made.